exercise3.js

```diff
@@ -119,17 +203,41 @@ console.log(mapObj.has({company : "TEKsystems"}));
 
 let ones = [1,11,111,1111,11111,111111,1111111,11111111,111111111,1111111111];
 //reverse the array, without modifying the ones array.
+for(let x = 0; x<ones.length/2; x++){
+    let temp = ones[x]
+    ones[x]=ones[ones.length-1-x];
+    ones[ones.length-x-1]=temp;
+}
```

Comment:
the problem here is, you have mutated the original array.  the instructions were to reverse the rarray WITHOUT modifying the existing array.